l3kernel/l3cctab.dtx

```diff
@@ -543,7 +543,7 @@
       {
         \seq_gpop:NNF \g_@@_unused_seq \l_@@_internal_a_tl
           { \@@_begin_aux: }
-        \exp_args:Nx \@@_chk_group_begin:n
+        \@@_chk_group_begin:e
```

Comment:
Why define a one-off `e` variant if the one-off `x`-variant wasn't necessary? The `\@@_chk_group_end:n` still uses `\exp_args:Ne`.

Reply:
Looking back over the code as a whole, I suspect we'd be better defining variants and only use 'raw' `\exp-args:N...` where we _have_ to, e.g. `\exp_arg:NV #1` where `#1` is something passed from higher up where we cannot be sure of the arg spec at all. I was planning to look at that systematically later: I did. a few of them, but they were not all `x`-type related and I decided part-way through to come back to this.